wrappers/node/package.json

```diff
@@ -1,96 +1,111 @@
+  "description": "Node wrapper for vcx",
+  "license": "Apache-2.0",
+  "typings": "dist/index.d.ts",
+  "directories": {
+    "test": "test",
+    "build": "dist",
+    "lib": "src"
+  },
+  "contributors": [
+    {
+      "name": "Hyperledger Indy Contributors",
+      "email": "hyperledger-indy@lists.hyperledger.org"
+    }
+  ],
+  "bugs": {
+    "url": "https://github.com/AbsaOSS/libvcx/issues"
+  },
+  "standard": {
+    "env": [
+      "mocha"
+    ]
+  },
+  "repository": {
+    "url": "https://github.com/AbsaOSS/libvcx.git",
+    "type": "git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "version": "0.8.0",
+  "dependencies": {
+    "@types/ffi-napi": "^2.4.1",
+    "@types/node": "^8.0.47",
+    "@types/ref-napi": "^1.4.0",
+    "@types/ref-struct-di": "^1.1.0",
+    "@types/uuid": "^8.3.0",
+    "@types/weak-napi": "^1.0.0",
+    "ffi-napi": "^2.4.7",
+    "fs-extra": "^4.0.2",
+    "lodash": "^4.17.11",
+    "ref-napi": "^1.4.3",
+    "ref-struct-di": "^1.1.0",
+    "uuid": "^8.3.0",
+    "weak-napi": "^1.0.3"
+  },
+  "scripts": {
+    "demo:notifyserver": "node notification-server.js",
+    "demo:alice": "node demo/alice.js",
+    "demo:faber": "node demo/faber.js",
+    "demo:faber:rev": "node demo/faber.js --revocation",
+    "demo:alice:pg": "node demo/alice.js --postgresql",
+    "demo:faber:pg": "node demo/faber.js --postgresql",
+    "demo:alice:sign": "node demo/alice-signature.js",
+    "demo:faber:verify": "node demo/faber-verify-signature.js",
+    "compile": "./node_modules/.bin/tsc -p ./tsconfig.json",
+    "clean": "rm -rf dist node_modules",
+    "lint": "./node_modules/.bin/tslint --type-check -c ./tslint.json -p ./tsconfig.json && ./node_modules/.bin/tslint --type-check -c ./test/tslint.json -p ./test/tsconfig.json",
+    "jslint:fix": "standard --fix",
+    "lint:demo": "standard demo/*",
+    "doc-gen": "./node_modules/.bin/typedoc --out doc --excludePrivate --excludeProtected --ignoreCompilerErrors src",
+    "test": "npm run test:connection && npm run test:credentialDef && npm run test:credential && npm run test:disclosedProof && npm run test:issuerCredential && npm run test:proof && npm run test:schema && npm run test:utils && npm run test:wallet && npm run test:ffi && npm run test:logging",
+    "test:aries": "npm run test:aries:connection && npm run test:aries:credentialDef && npm run test:aries:credential && npm run test:aries:disclosedProof && npm run test:aries:issuerCredential && npm run test:aries:proof && npm run test:aries:schema && npm run test:aries:utils && npm run test:aries:wallet && npm run test:ffi && npm run test:logging",
+    "test:aries:connection": "      DISALLOW_V1_CONNECTION=true LIBVCX_FFI_ARCHITECTURE=x64 TS_NODE_PROJECT=./test/tsconfig.json NODE_ENV=test RUST_BACKTRACE=full ./node_modules/.bin/mocha --timeout 10000 -gc --expose-gc --exit --recursive --use_strict --require ts-node/register ./test/suite1/ariesvcx-connection.test.ts",
```

Comment:
I guess this can't be done more concisely... Why not set `LIBVCX_FFI_ARCHITECTURE` `x64` by default? Can you please explain where is `DISALLOW_V1_CONNECTION` used?

Reply:
- Good catch, `DISALLOW_V1_CONNECTION` was leaking here from another PR, removed now.
- Originally `LIBVCX_FFI_ARCHITECTURE` was made to default to x86 for backwards compatibility, but yeah I think it makes sense to switch default to x64. 👍